### PR TITLE
Create master credentials only once.

### PR DESCRIPTION
**Pull request by rgooch**

### CODE REVIEW

File: broker/aws/impl.go [hunk 1]
```diff
@@ -126,6 +127,27 @@ func (b *Broker) accountHumanNameFromName(accountName string) (string, error) {
...
+	if err != nil {
+		return err
+	}
+	if masterSession == nil {
+		return errors.New("masterSession == nil")
+	}
+	b.masterStsClient = sts.New(masterSession)
+	b.masterStsRegion = region
+	b.isUnsealedChannel <- nil
+	return nil
```
Comment (cviecco) [inline broker/aws/impl.go:148]: it was my understanding that even these sessions expire... so thi may stop working after 8 hours... have you tested this in a long running service?

**Review by cviecco — COMMENTED** [review]
Can this really be cached for days?

File: broker/aws/impl.go (hunk 1 above)
Comment (rgooch) [reply broker/aws/impl.go:148]: I've been running this over the weekend.

Comment (rgooch) [reply broker/aws/impl.go:148]: Instances were up for over 24 hours. I tested both master role (EC2 role) and static credentials master.

Comment (rgooch) [reply broker/aws/impl.go:148]: For EC2 it's actually a credentials provider, which under the covers regenerates them when they are close to expiring.

**Review by cviecco — APPROVED** [review]

**Pull request merged**